server/crud-service.js

```diff
@@ -32,17 +29,16 @@ setInterval(function () {
 }, 1000);
 
 service.on('request', (rid, key, payload, handler) => {
-    payload = JSON.parse(payload);
-    if (payload.action === 'put') {
-        console.log(payload);
-        wl.put(payload.text, {}, (err, hash) => {
-            if (err) throw err;
-            handler.reply(null, hash)
-        })
-    } else {
-        wl.get(payload.hash, {}, (err, data) => {
-            if (err) throw err;
-            handler.reply(null, data)
-        })
-    }
+  if (typeof payload === 'string') payload = JSON.parse(payload);
```

Comment:
when someone sends invalid json it will crash the service. example

```js
const payload = 'foo'
JSON.parse(payload)
```

please add a try/catch here